dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -210,6 +210,9 @@ def main(args):
 
     scheduler_workers = client.run_on_scheduler(get_scheduler_workers)
     n_workers = len(scheduler_workers)
+    client.wait_for_workers(n_workers)
+
+    all_to_all(client)
```

Comment:
Maybe we can add a flag to enable this? I would currently prefer it disabled by default, just to be closer to what's expected from a regular user's code today. Although, this may be a fix to the issue that @quasiben and I were debugging today, still to be confirmed.